src/validators/restrictionType.validator.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ *
+ * Copyright 2018-present NEM
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+import { ExpectedError, ValidationContext, Validator } from 'clime';
+
+export class MosaicRestrictionTypeValidator implements Validator<string> {
+    validate(value: string, context?: ValidationContext): void {
+        const limitValue = ['NONE', 'EQ', 'NE', 'LT', 'LE', 'GT', 'GE'];
```

Comment:
Same here, we could use the nem2-sdk ``MosaicRestrictionType`` enum as we are doing for ``NetworkType``.

Example: https://github.com/nemtech/nem2-cli/pull/175/files#diff-8da91dc84947c2dbbc018cc9623a7755

Reply:
@dgarcia360 
I don't know how to resolve it. Now the code is like that
 [https://github.com/nemtech/nem2-cli/pull/109/files#diff-5992909fcb4b993ded55b71d85e3b2ae](https://github.com/nemtech/nem2-cli/pull/109/files#diff-5992909fcb4b993ded55b71d85e3b2ae). 
And I have no idea to make it better.

Reply:
How about:

`` if (!value in MosaicRestrictionType) throw new ExpectedError('Wrong mosaic restriction type');``